frontend/packages/dev-console/src/components/topology/components/nodes/PodSet.tsx

```diff
@@ -51,6 +69,16 @@ const PodSet: React.FC<PodSetProps> = ({ size, data, x = 0, y = 0 }) => {
         outerRadius={podStatusOuterRadius}
         data={completedDeploymentData}
         size={size}
+        subTitle={showPodCount && subTitle}
+        {...showPodCount &&
+          !accessAllowed && {
+            subTitleComponent: <ChartLabel style={{ fontSize: '14px' }} />,
```

Comment:
This looks like duplicate from `PodRing`. Same goes for `titleComponent`.
Why aren't these provided in `PodStatus` ?

Reply:
refactored this a bit so this is more readable. There are many conditions that the label depends on but now the utility itself will also return the title and subtitle component